szperacz/run.py

```diff
@@ -18,4 +18,13 @@
 
     files = discover_files(path)
 
-    print(files)
+
+def log_to_console(files):
+    print(f'Images found: {len(files)}\n---')
+    for i in files:
+        print(f'{i}')
+
+    return '---'
+
+
+print(log_to_console(files))
```

Comment:
1. Function body looks fine, but it is misplaced. Please move `log_to_console` definition above the main entry of the program - above `if __name__ == __main__`. It is a good habit to put definition of the function above the call. In Python it does not matter that much, but this way of doing things is more natural for human reading and in fact is enforced by other languages.

2. `print(log_to_console(files))` is wrongly placed in terms of indentation. It should be a part of main script execution, not some lone call after the main path. Please add one indentation (1 tab or 4 spaces).